ganttproject/src/biz/ganttproject/impex/csv/XlsReaderImpl.java

```diff
@@ -53,11 +60,50 @@ public void close() throws IOException {
 
   @Override
   public Iterator<SpreadsheetRecord> iterator() {
-    return Iterators.transform(myBook.getSheetAt(0).iterator(), (input) -> new XlsRecordImpl(getCellValues(input), myHeaders));
+    Iterator<SpreadsheetRecord> iterator = null;
+
+    if(myBook.getNumberOfSheets() > 1) {
+      Iterator<SpreadsheetRecord> it = null;
+      for (int i = 0; i < myBook.getNumberOfSheets(); ++i) {
+        Sheet sheet = myBook.getSheetAt(i);
+        if (i < myBook.getNumberOfSheets() - 1) {
+          int lastRow = sheet.getPhysicalNumberOfRows();
+          sheet.createRow(lastRow + 1);
```

Comment:
This looks a little bit weird: why do we create rows when reading? Even if it makes no changes on the disk (I believe it doesn't) it worth adding a comment about the purpose of these two rows.

Reply:
That is because of the general CSV-Reader which expects to have two empty lines between tables e.g. tasks and ressources.